packages/transformers/vue/src/VueTransformer.js

```diff
@@ -70,7 +70,7 @@ export default (new Transformer({
     let baseId = md5FromObject({
       filePath: asset.filePath,
     }).slice(-6);
-    let scopeId = 'data-v-' + baseId;
+    let scopeId = baseId;
```

Comment:
You may consider removing this/renaming variables for clarity

Reply:
I considered that but I thought that keeping `scopeId` provided more clarity than just `id`, and kept changes minimal.